README.md

```diff
@@ -1,87 +1,237 @@
+options.
+
+## Compatibility
+
+Projects maintained by the unified collective are compatible with all maintained
+versions of Node.js.
+As of now, that is Node.js 12.20+, 14.14+, and 16.0+.
+Our projects sometimes work with older versions, but this is not guaranteed.
+
+This plugin works with `unified` version 10+.
+
+## Related
+
+*   [unified][]
+    — create pipeline for working with syntax trees
+*   [unified-args][]
+    — create a CLI for a unified pipeline
+
+## Contribute
+
+See [`contributing.md`][contributing] in [`unified/.github`][health] for ways
+to get started.
+See [`support.md`][support] for ways to get help.
+
+This project has a [code of conduct][coc].
+By interacting with this repository, organization, or community you agree to
+abide by its terms.
+
+## License
+
+[MIT][license] © [Titus Wormer][author]
+
+<!-- Definitions -->
+
+[build-badge]: https://github.com/unifiedjs/unified-language-server/workflows/main/badge.svg
+
+[build]: https://github.com/unifiedjs/unified-language-server/actions
+
+[coverage-badge]: https://img.shields.io/codecov/c/github/unifiedjs/unified-language-server.svg
+
+[coverage]: https://codecov.io/github/unifiedjs/unified-language-server
+
+[downloads-badge]: https://img.shields.io/npm/dm/unified-language-server.svg
+
+[downloads]: https://www.npmjs.com/package/unified-language-server
+
+[size-badge]: https://img.shields.io/bundlephobia/minzip/unified-language-server.svg
+
+[size]: https://bundlephobia.com/result?p=unified-language-server
+
+[sponsors-badge]: https://opencollective.com/unified/sponsors/badge.svg
+
+[backers-badge]: https://opencollective.com/unified/backers/badge.svg
+
+[collective]: https://opencollective.com/unified
+
+[chat-badge]: https://img.shields.io/badge/chat-discussions-success.svg
+
+[chat]: https://github.com/unifiedjs/rehype/discussions
+
+[npm]: https://docs.npmjs.com/cli/install
+
+[health]: https://github.com/unifiedjs/.github
+
+[contributing]: https://github.com/unifiedjs/.github/blob/HEAD/contributing.md
+
+[support]: https://github.com/unifiedjs/.github/blob/HEAD/support.md
+
+[coc]: https://github.com/unifiedjs/.github/blob/HEAD/code-of-conduct.md
+
+[language server]: https://microsoft.github.io/language-server-protocol/
+
+[license]: LICENSE.txt
+
+[author]: https://wooorm.com
+
+[redot-language-server]: https://github.com/redotjs/redot-language-server
 
-So, for a markdown file:
+[rehype-language-server]: https://github.com/redotjs/rehype-language-server
 
-1.  because we launched it with `--parser=remark-parse`, it finds the setting
-    with the same name
-2.  applies all the plugins:
-    1.  `remark-preset-lint-markdown-style-guide` checks for markdown usage
-    2.  `remark-retext` extracts the texts from markdown
-    3.  `retext-profanities` about usage of profanity words
-    4.  `retext-spell` does spellcheck
+[remark-language-server]: https://github.com/redotjs/remark-language-server
```

Comment:
These language servers don’t actually exist yet, but I imagine they will be implemented soon after this PR is released.

Reply:
Maybe add a comment that they are coming soon instead?